src/libpmemfile-core/inode.c

```diff
@@ -80,6 +80,8 @@ pmfi_path(struct pmemfile_vinode *vinode)
 struct pmemfile_vinode *
 vinode_ref(PMEMfilepool *pfp, struct pmemfile_vinode *vinode)
 {
+	(void) pfp; /* XXX: could this ever be needed? */
```

Comment:
Yes, please remove this comment (and the one in inode_free). Other than that it LGTM.